### ISSUE TITLE
PM UI Icon Status could be indicated with an Overlay on Default Icon

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#10122]
A recent PR https://github.com/NuGet/NuGet.Client/pull/3702 by @rrelyea adds Statuses ([IconBitmapStatus](https://github.com/NuGet/NuGet.Client/blob/fc7040a568c576511d0546209666d12e0c1503ba/src/NuGet.Clients/NuGet.PackageManagement.UI/Models/IconBitmapStatus.cs)) for why an icon failed to load.
It may be worth considering adding some overlays to the Default Package  icons, so it's clear why an icon failed to load (is it because the client is offline? Issue with the package itself? Or just no icon for this package?)

For accessibility and clarity, adding a tooltip explaining the status would be warranted.

### Discussion prompting this issue
https://github.com/NuGet/NuGet.Client/pull/3702#discussion_r504884883
https://github.com/NuGet/NuGet.Client/pull/3702#discussion_r504891231

I proposed on a whim something like these for Network issues:
![image](https://user-images.githubusercontent.com/49205731/96029726-0da07680-0e29-11eb-95f2-ed6c6007649e.png)
![image](https://user-images.githubusercontent.com/49205731/96029902-49d3d700-0e29-11eb-9bbc-a0c8349f6e25.png)
